website/addons/dryad/static/node-cfg.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var $ = require('jquery');
+require('./dryad-node-config.js');
+var bootbox = require('bootbox');
+var AddonHelper = require('js/addonHelper');
+
+
+    $(document).ready(function() {
+
+        $('#dryadsubmitkey').on('click', function() {
+        	var doi = $('#dryaddoitext').val();
+
+            var check_url = $(dryad_check_url).val();
+            var add_url = $(dryad_add_url).val();
+        	var doi_response = $.ajax({
+	    		type: "GET",   
+    			url: check_url,   
+                data: {'doi' : doi},
+			    success : function(response) {
```

Comment:
It might make sense to convert some of the views that don't need a rendered HTML response to use the `json_renderer`, `check_dryad_doi` being a good example of this. 

The only changes required would be changing the renderer in your routes.py and doing
`if (response.result === true)` here instead.